packages/icons/src/components/Icon.tsx

```diff
@@ -3,15 +3,15 @@ import React, { useMemo } from 'react';
 import { Canvas, ImageSVG, useSVG, Group, Skia, BlendMode } from '@shopify/react-native-skia';
 
 import { useNativeStyles } from '@trezor/styles';
-import { Color } from '@trezor/theme';
+import { Color, CSSColor } from '@trezor/theme';
 
 import { IconName, icons } from '../icons';
 
 type IconProps = {
     name: IconName;
     size?: IconSize;
     customSize?: number;
-    color?: Color;
+    color?: Color | CSSColor;
```

Comment:
Why do we need this? We should probably never use color that is not our theme pallete.

Reply:
Ideally, we should not use any colour out of our colour theme. But this new notification design is using "inverted" theme. That means that if the app theme is set to "standard" the displayed notification will use colours from the dark theme and vice versa. 

For this reason, I need to extract the hex value of the dark theme yellow to the white colour theme.

Reply:
fixed in 0dfad91